### ISSUE TITLE
AutoCrafting bug

### ISSUE THREAD

**Issue by Fragarah** [AppliedEnergistics/Applied-Energistics-2#2979]
I'm not a programmer but I've got strange bug recently. The system shows that I can craft pure fluix crystal but when I do it nothing happens. The pattern is normal seed into crystal but when I removed it from interface system still shows that I can craft it but there is no item required only " to craft:'number' " nothing will ever happen but it ends up with my crafting computer getting constantly stuck.
Few days before I got another problem. I placed formation plane but then I resigned from a build and removed everything but every item I tried to put into the system was dropped under block where a plane was. Replacing Controller solved this. But this time I rebuilded whole controller and pure crystal autocrafting setup and nothing there is still a bug and only for fluix pure crystals.

- Minecraft Version: 1,7,10
- AE2 Version:  rv2 stable
- Forge Version: 10,134,1558

**Comment by xenxero1** [comment]
Are you sure you didn't put the crafting pattern in an interface next to an assembler or something? It might be trying to register this as a normal crafting recipe. Try clearing all items in your molecular assemblers and check the pattern storages in the interfaces. Make sure the crafting computers have no active builds as well.

I'm not sure what you mean by formation plane either.

**Comment by Fragarah** [comment]
There are no assemblers yet cuz I'm building a totally new network for a new base so I got only:
 20 block controller 
few terminals
 one drive with some item for building purpose
This pure crystal build it actually the first thing I'm doing here so I'm 100% sure it was the only pattern for it and even when I put pattern back there into interface buggy crafting remained

**Comment by xenxero1** [comment]
So, is the interface hooked into something, like a hopper that will drop it into a pool of water/crystal growth system? Or is the pattern just in a lone interface?

**Comment by Fragarah** [comment]
It's interface with patterns for all pure and fluix crystals feeding into sub net interface with formation plane then another subnet annihilation plane into me chest with partitioned cell for pure and fluix and lastly export bus into main network like this: link with screen
[http://screenshooter.net/100299497/oyroxib](url)

**Comment by yueh** [comment]
We no longer support 1.7.10 at this point.

Should you be able to confirm that this issue still apply to the most recent version, please update the issue and we will reopen it.

**Comment by CrossVas** [comment]
If the problem is still relevant.

I noticed you have 2 subAE system. Why? I don't know.
Both of subAE sys. aren't online (if look at screenshot). 

I try to reproduce Your build with some modifications.
I got this:
![image](https://user-images.githubusercontent.com/25727222/28916567-4f339f0e-784b-11e7-8d36-fe2f33ad2f26.png)
![image](https://user-images.githubusercontent.com/25727222/28916595-611cd33e-784b-11e7-8754-d5ab836e8615.png)
- Used one subAE sys.;
- Used Vacuum Chest (from Ender IO) with filter to collect pure crystals instead of Annihilation Plane.
  !Annihilation Plane don't wait when the seed grows to crystal.
